Shopist/Shopist/AppDelegate.swift

```diff
@@ -9,7 +9,7 @@ import Datadog
 
 fileprivate(set) var logger: Logger!
 
-let appConfig = ExampleAppConfig(serviceName: "ios-sdk-example-app")
+let appConfig = ExampleAppConfig(serviceName: "ios-sdk-shopist-app-debug")
```

Comment:
`debug` has the environment meaning. If we want it, then it should be used for: `Datadog.Configuration.builderUsing(clientToken:environment)`.